extensions/ui/org.eclipse.smarthome.ui.paper/web-src/js/controllers/controller.configurableService.dialog.js

```diff
@@ -29,6 +29,27 @@ configurableServiceDialogController.controller('ConfigurableServiceDialogControl
         }).then(applyDefaults(configDescriptionURI));
     }
 
+    var addESHServiceContextParameter = function(configDescription) {
+        var found = false;
+        angular.forEach(configDescription.parameters, function(parameter) {
+            if (parameter.name === 'esh.servicecontext') {
+                found = true;
+            }
+        })
+        if (!found) {
+            // the service context field is the name of this configuration to help the user to distinguish between
```

Comment:
I think you mis-understood me (and I though by now you know how I think about comments in code... 😉 ) 

With a *description*  for a *user*  I was referring to something in the sense of [ConfigDescriptionParameterDTO.description](https://github.com/eclipse/smarthome/blob/4204ce06bb28c28e5f711e720f87ef83beff2e27/bundles/config/org.eclipse.smarthome.config.core/src/main/java/org/eclipse/smarthome/config/core/dto/ConfigDescriptionParameterDTO.java#L33) like you have added [here](https://github.com/eclipse/smarthome/pull/4847/files#diff-d3ca81dfa99af7ad121aea1b1ece5c52R10), i.e. 

```js
var eshServiceContextParameter = {
   ...
   description : 'bla',
   ...
}
```

This is what the user currently sees:

![image](https://user-images.githubusercontent.com/2592443/34878280-ecc3e1f6-f7a9-11e7-936a-d9764c59ada6.png)

and sorry, but that really doesn't give any hint about what that is meant to be!

Reply:
Indeed there was a misunderstanding.

I have added the description to the parameter but still think that the comment is also useful.

Note: javascript code is harder to read than java :) And the developer might wonder why we add that parameter there.